.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,5 +21,6 @@ While we look over every pull request, we maintain a focus on this project's cur
 
 <!-- Put an 'x' in the boxes that apply. -->
 
+- [ ] I have tested my changes on our frc robot's hardware. 
```

Comment:
"frc" should be capitalized. Or (IMO better option) just "I have tested my changes on real hardware."